app/views/users/show.html.erb

```diff
@@ -1,13 +1,92 @@
 <% provide(:title, @user.first_name) %>
-<div class="container mb-10">
-  <div class="row">
-    <aside class="col-md-4">
-      <section class="user_info">
-        <h1>
-          <%= gravatar_for @user %>
-          <%= @user.first_name %>
-        </h1>
-      </section>
-    </aside>
+<div class="container mb-50 mt-50">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-4">
+        <div class="panel panel-default">
+          <div class="panel-heading mb-50">
+            <h4><%= t "views.users.show.profile"%></h4>
+          </div>
+          <div class="panel-body">
+            <div class="box box-info">
+              <div class="box-body">
+                <div class="row">
+                  <div class="col-sm-12">
+                    <div class="center">
+                      <%= image_tag avatar_img(current_user.avatar_url), id: "profile-image1", class: "img-circle" %>
+                      <%= file_field_tag "profile-image-upload", class: "hidden" %>
```

Comment:
chú điều hướng 1 cái label cho for vào cái này đi là được mà, cần gì hidden

Reply:
chỗ này em muốn là nhấn vào cái ảnh thì nó bật lên cửa sổ chọn ấy ạ
![Screenshot from 2020-01-19 14-59-29](https://user-images.githubusercontent.com/57735681/72677202-63c7cf80-3acc-11ea-8174-0ee6519d7dcf.png)

Reply:
dùng label với attribute for="some_id" là được, nghiên cứu thử đi, còn pull nay thì ko cần sửa ngay đâu